db.go

```diff
@@ -79,6 +80,10 @@ type Options struct {
 	// Overlapping blocks are allowed if AllowOverlappingBlocks is true.
 	// This in-turn enables vertical compaction and vertical query merge.
 	AllowOverlappingBlocks bool
+
+	// ReadOnly disables all mutating actions.
+	// Useful for opening the same db more than once.
+	ReadOnly bool
```

Comment:
Have we considered totally different constructor instead returning limited TSDB interface ?